app/models/concerns/email_regex.rb

```diff
@@ -0,0 +1,28 @@
+module EmailRegex
+  extend ActiveSupport::Concern
+
+  def self.match(value, match_rule, _should_match, check_mode = false)
```

Comment:
Two things:
- Why is there an argument `_should_match` which is never used?
- We should use keyword arguments. A rule of thumb: More than two arguments should always be keyword arguments. This increases the readability a lot.

Reply:
yeah. i was using positional arguments and i think keyword arguments will increase readablity. _should_match came from the original method which i forgot to exclude from earlier.